### PR TITLE
Netlify badge

### PR DESCRIPTION
**Pull request by macrigiuseppe**
Added netlify badge to readme page
Signed-off-by: Giuseppe Macri <gmacri@uber.com>

Implements: #639

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,9 @@
   <a href='https://app.fossa.com/projects/custom%2B4458%2Fgithub.com%2Fkeplergl%2Fkepler.gl?ref=badge_shield'>
     <img src='https://app.fossa.com/api/projects/custom%2B4458%2Fgithub.com%2Fkeplergl%2Fkepler.gl.svg?type=shield' alt='Fossa' />
   </a>
+  <a href="https://app.netlify.com/sites/keplergl/deploys" alt="Netlify Status">
+    <img src="https://api.netlify.com/api/v1/badges/0c9b895c-acd0-43fd-8af7-fe960181b686/deploy-status" />
```
Comment (heshan0131) [inline README.md]: You can use `shield.io` to build netlify badge consistent with our other badge style. replace ths image.src with
`https://img.shields.io/endpoint.svg?url=https%3A%2F%2Fdeveloper.oswaldlabs.com%2Fnetlify-status%2F0c9b895c-acd0-43fd-8af7-fe960181b686?style=flat`

More detail in this thread:
https://github.com/badges/shields/issues/3129#issuecomment-470032754

Comment (macrigiuseppe) [reply README.md]: Done

**Comment by heshan0131** [comment]
Can you also fix github stars with
```
  <a href="https://github.com/keplergl/kepler.gl">
    <img src="https://img.shields.io/github/stars/keplergl/kepler.gl.svg?style=flat" alt="stars" />
  </a>
```

**Comment by macrigiuseppe** [comment]
> Can you also fix github stars with
> 
> ```
>   <a href="https://github.com/keplergl/kepler.gl">
>     stars
>   </a>
> ```

@heshan0131 yep done!

**Pull request merged**